src/App/_tests_/articles/article.test.js

```diff
@@ -0,0 +1,217 @@
+import React from 'react';
+import thunk from 'redux-thunk';
+import promiseMiddleware from 'redux-promise-middleware';
+import configureStore from 'redux-mock-store';
+import { MemoryRouter } from 'react-router-dom';
+import { mount } from '../../../enzyme';
+import { Article } from '../../Components/Articles/getArticles/Article';
+import countRating from '../../../Helpers/countRating';
+
+
+const middlewares = [thunk, promiseMiddleware];
+const mockStore = configureStore(middlewares);
+
+
+describe('<articles />', () => {
+  describe('render()', () => {
+    it('should give rating', () => {
```

Comment:
@Ramadhan0 edit the test descriptions. Are you testing that it `should give rating` or that it `should display the right rating`?